src/occt_wrapper/CMakeLists.txt

```diff
@@ -19,35 +19,23 @@ include(GenerateExportHeader)
 
 generate_export_header(OCCTWrapper)
 
-find_package(OpenCASCADE 7.6.1 REQUIRED)
+find_package(OpenCASCADE REQUIRED)
```

Comment:
I'm afraid this will conflict with expectations from https://github.com/prusa3d/PrusaSlicer/commit/c6a02106fd1d3caa9a48a6b7c2bdd04546b24485.

Specifically for Fedora I filed https://bugzilla.redhat.com/show_bug.cgi?id=2333653 as the issue does not seem to be fixed in OCCT 2.8.1 which we have there.

Reply:
Fedora already has OpenCascade 7.8.1. Looking at https://tracker.dev.opencascade.org/ I'm not able to find a bug report about it.

I wonder if this has been reported upstream.

Reply:
The very same question popped into my head, I asked about it in https://github.com/prusa3d/PrusaSlicer/issues/12122#issuecomment-2558141616.